README.md

```diff
@@ -151,6 +154,7 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
 | <a href="https://craftwork.gumroad.com/l/ZWiR"><img width='700' src='https://user-images.githubusercontent.com/74038190/184810212-6b2ee06f-81a7-413c-b967-a899e9b7b56c.png' alt="Control_free_illustrations" /></a> | [Control](https://craftwork.gumroad.com/l/ZWiR)  <br><br> ✔ Free <br><br>Format <br> • PNG<br> • Adobe Illustrator<br><br> ✌️Author <br> Craftwork.design                                                                                             |
 | <a href="https://www.getillustrations.com/illustration-pack/download-free-daily-illustrations"><img width='700' src='https://user-images.githubusercontent.com/74038190/184478648-65d44155-3b84-482d-adde-9878ae1efdfa.png' alt="Daily 50_free_getillustrations" /></a> | [Daily 50](https://www.getillustrations.com/illustration-pack/download-free-daily-illustrations)  <br><br> ✔ Free <br><br>Format <br> • Adobe XD<br>• Figma<br>• Illustrator<br>• Sketch<br><br> ✌️Author <br>Getillustrations                                                     |
 | <a href="https://delesign.com/free-designs/graphics/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184916250-3f55d22b-e96e-45d2-93bc-13bdfe07f24d.png' alt="Delesign_free_illustrations" /></a> | [Delesign](https://delesign.com/free-designs/graphics/)  <br><br> ✔ Free <br><br>Format <br> • SVG<br>• PNG<br><br> ✌️Author <br> Delesign                                                                                |
+| <a href="https://avatars.dicebear.com"><img width='700' src='https://raw.githubusercontent.com/Sn4pe/Awesome-Illustrations-4Projects/main/assets/dicebear-avatar.png' alt="DiceBear_Avatars" /></a> | [DiceBear Avatars](https://avatars.dicebear.com) <br><br> ✔ Free <br><br>Format <br> • API<br>• SVG<br><br>                    |
```

Comment:
- Include the Author: [Florian Körner](https://github.com/FlorianKoerner)

- Include the formats: [PNG & JPG](https://avatars.dicebear.com/integrations/cli#create-avatar-in-png-or-jpg-format) 

- Change the alt text to `DiceBear_Avatars__free_illustrations`

Reply:
@Sn4pe 
You have not included the author's name with the latest commit.